main.tf

```diff
@@ -350,11 +350,7 @@ module "ranchhand" {
   source = "github.com/dominodatalab/ranchhand?ref=v0.3.1"
 
   // If for some bizarre reason you mix public/private hosts, this won't work
-  node_ips = formatlist(
-          (aws_instance.this[0].public_ip != "" ? "%s:%s" : "%[2]s"),
-          aws_instance.this.*.public_ip,
-          aws_instance.this.*.private_ip,
-        )
+  node_ips = coalescelist(aws_instance.this.*.public_ip, aws_instance.this.*.private_ip)
```

Comment:
This still needs to be formed as a `public:private` string, OR we should just deprecate installing rancher into private clusters. We don't do it, we're not supporting external users of this tf module that do it, so maybe just set node_ips to private_ip statically?

Reply:
Done :+1: